common4j/src/main/com/microsoft/identity/common/java/logging/Logger.java

```diff
@@ -108,6 +108,11 @@ public enum LogLevel {
          * Undefined. Should be used in test only.
          */
         UNDEFINED,
+
+        /**
+         * No logs should be emitted
+         */
+        NO_LOG
```

Comment:
>NO_LOG [](http://example.com/codeflow?start=8&length=6)

should this be first entry in the enum to ensure e1.compareTo.e2() checks work as expected?

Reply:
The comparison would not fail. But I moved it to first entry as I need to order it from most restrictive to least restrictive log level.

Reply:
on both files?

Reply:
Well, since you moved it this means that each value in this file now has got a new ordinal value. I suggest checking with OneAuth that they are not relying on the ordinal value.

Reply:
Should be fine. These are the uses of Common's Logger values here:
https://office.visualstudio.com/OneAuth/_git/OneAuth?path=/sources/public/android/java/com/microsoft/authentication/OneAuth.java&version=GBdev&line=647&lineEnd=648&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

and

https://office.visualstudio.com/OneAuth/_git/OneAuth?path=/sources/public/android/java/com/microsoft/authentication/OneAuth.java&version=GBdev&line=698&lineEnd=699&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

Both of these locations don't use the ordinal values of Common's LogLevel enums